Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -715,7 +715,8 @@ class RenderStyle {
 
     inline ContentVisibility contentVisibility() const;
 
-    inline std::optional<ContentVisibility> skippedContentReason() const;
+    inline ContentVisibility effectiveContentVisibility() const;
+    inline bool hasSkippedContent() const;
```

Comment:
Maybe a comment explaining the return values would be useful. It's not super obvious what the first one returns in the `auto` case.

Reply:
True, I added comments.